src/docker-osx-dev

```diff
@@ -409,7 +409,12 @@ function configure_docker_machine {
   DOCKER_MACHINE_DRIVER_NAME=$(inspect_docker_machine "{{.DriverName}}")
 
   DOCKER_HOST_SSH_URL="$DOCKER_HOST_USER@$DOCKER_HOST_IP"
-  DOCKER_HOST_SSH_KEY="$DOCKER_MACHINE_STORE_PATH/id_rsa"
+  DOCKER_HOST_SSH_KEY="$DOCKER_MACHINE_STORE_PATH/machines/$DOCKER_MACHINE_NAME/id_rsa"
+
+  if [[ ! -f $DOCKER_HOST_SSH_KEY ]]; then
```

Comment:
That might one breaking change on docker-machine 0.5.

I am on 0.4.1 and my `{{.StorePath}}` is like `$HOME/.docker/machine/machines/$DOCKER_MACHINE_NAME`. So `DOCKER_HOST_SSH_KEY="$DOCKER_MACHINE_STORE_PATH/id_rsa"`, seems correct for 0.4.1.